test/src/main/java/org/apache/accumulo/test/ComprehensiveIT.java

```diff
@@ -133,6 +153,51 @@ public static void teardown() {
     SharedMiniClusterBase.stopMiniCluster();
   }
 
+  @Test
+  public void testEventualScan() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      String tableName = getUniqueNames(1)[0];
+      client.tableOperations().create(tableName);
+
+      try (Scanner scanner = client.createScanner(tableName, Authorizations.EMPTY)) {
+
+        // Add data to the table
+        try (BatchWriter writer = client.createBatchWriter(tableName, new BatchWriterConfig())) {
+          Mutation m = new Mutation("row1");
+          m.put("cf", "cq", "value1");
+          writer.addMutation(m);
+        }
+
+        // Run initial scan with eventual consistency
```

Comment:
I would not think that this eventual scan would consistently find the data as you are not flushing the table after inserting the data into it. If you have not seen them already, take a look at `ScanServerIT.testScan` and `ScanServerIT.testBatchScan`. It might be sufficient to copy them here or invoke them from here if that actually works.